### PR TITLE
Loki: Run instant query only when doing metric query

### PR DESCRIPTION
**Pull request by aocenas**
In explore we are running both instant and range query similar to prometheus but we should not have run it for logs queries where it does not make sense and we do not show the result anyway.

### CODE REVIEW

File: public/app/plugins/datasource/loki/datasource.test.ts [hunk 1]
```diff
@@ -36,33 +36,33 @@ const timeSrvStub = {
   }),
 };
 
+const testResponse: FetchResponse<LokiResponse> = {
```
Comment (aocenas) [inline public/app/plugins/datasource/loki/datasource.test.ts:39]: Just moved.

**Comment by aocenas** [comment]
@ivanahuckova do you know if there was an issue for this?

File: public/app/plugins/datasource/loki/datasource.ts [hunk 2]
```diff
@@ -99,7 +99,9 @@ export class LokiDatasource extends DataSourceApi<LokiQuery, LokiOptions> {
       }));
 
     for (const target of filteredTargets) {
-      if (options.app === CoreApp.Explore) {
+      // In explore we want to show instant query in a table under the graph panel to mimic behaviour of prometheus.
+      // We don't want to do that in dashboards though as user would have to pick the correct data frame.
```
Comment (ivanahuckova) [inline public/app/plugins/datasource/loki/datasource.ts:103]:
```suggestion
      // In explore we want to show result of metrics instant query in a table under the graph panel to mimic behaviour of prometheus.
      // We don't want to do that in dashboards though as user would have to pick the correct data frame.
```

Just a nit suggestion

**Review by ivanahuckova — APPROVED** [review]
Nice!

**Review by Elfo404 — APPROVED** [review]
LGTM 🚀

**Comment by ivanahuckova** [comment]
>@ivanahuckova do you know if there was an issue for this? 

I see that you have closed this already, but there weren't any issue for this. We have fixed that issue by not running instant queries in dashbaords.

**Comment by kylebrandt** [comment]
Removed cp label because it is in beta2, and see this in the v7.3.x branch already

**Pull request merged**